client/src/app/pages/controls/stakeholders/stakeholders.tsx

```diff
@@ -284,16 +285,7 @@ export const Stakeholders: React.FC = () => {
                         rowIndex={rowIndex}
                       >
                         <Td
-                          expand={{
-                            rowIndex,
-                            isExpanded: isCellExpanded(stakeholder),
-                            onToggle: () =>
-                              setCellExpanded({
-                                item: stakeholder,
-                                isExpanding: !isCellExpanded(stakeholder),
-                              }),
-                            expandId: "composable-expandable-example",
-                          }}
+                          {...getExpandTdProps({ item: stakeholder, rowIndex })}
```

Comment:
Can we rename it to `getSingleExpandTdProps` to distinguish it from the compound expand version?